cantusdb/csv/merge.py

```diff
@@ -0,0 +1,46 @@
+"""
+Merge all raw Cantus CSV into one Dataframe and export to a new CSV
+"""
+
+import os
+import pandas as pd
+
+INPUT_PATH = os.path.join(os.path.dirname(__file__), "../data/raw")
+OUTPUT_PATH = os.path.join(os.path.dirname(__file__), "../data/cantus.csv")
+MAPPING_PATH = os.path.join(os.path.dirname(__file__), "../data/mappings")
+
+# Create an empty list to store individual DataFrames
+dfs = []
+genre_mappings = pd.read_csv(
+    os.path.join(MAPPING_PATH, "genres.tsv"), sep="\t", header=0, index_col=False
+)
+service_mappings = pd.read_csv(
+    os.path.join(MAPPING_PATH, "services.tsv"), sep="\t", header=0, index_col=False
+)
+genre_mappings = dict(zip(genre_mappings["Genre"], genre_mappings["Description"]))
+service_mappings = dict(zip(service_mappings["Service"], service_mappings["Description"]))
+
+# Iterate over all files in the directory
+for filename in os.listdir(INPUT_PATH):
+    # Construct the full file path
+    file_path = os.path.join(INPUT_PATH, filename)
+
+    # Check if the file is a CSV file
+    if filename.endswith(".csv"):
+        # Read the CSV file into a DataFrame
```

Comment:
Could this be a place to use glob module?

Reply:
Why are we preferring glob instead of os.listdir?